microsoft/electionguard
# Final verifier for college park

[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
*Link your PR to an issue*

Fixes #___

### Description
*Please describe your pull request.*

### Testing
1. Review content page for College Park election for proper content
2. Ensure link to new requirements doc is valid
3. Ensure old requirements doc is removed
